ScreenOrientationManager/ScreenOrientationManager.py

```diff
@@ -107,6 +108,7 @@ def on_click(self, widget):
                          str(self.display_check.get_active()))
             if label.lower() == "left":
                 self.rotate("l")
+                subprocess.Popen['twofing', '/dev/input/event3']
```

Comment:
Isn't this specific to devices with a touchscreen?

Reply:
Sorry, it should not be there. It was specific to my touch device. I forced pushed a new commit.